app/main.py

```diff
@@ -12,7 +12,7 @@ def format_single_linter_file(file_path: str, errors: list) -> dict:
     return {
         "errors": [format_linter_error(item) for item in errors],
         "path": file_path,
-        "status": ("passed" if len(errors) == 0 else "failed"),
+        "status": "passed" if not errors else "failed",
```

Comment:
it's easier to read "failed" if errors else "passed". Try to simplify if statements if possible :)